firmware/config/engines/me7pnp.h

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
this file should be taken care by https://github.com/rusefi/rusefi/commit/d8747359af182d28b02a5a69329d66b435caea14